synchro.go

```diff
@@ -90,7 +90,7 @@ func Parse[T TimeZone](layout, value string) (Time[T], error) {
 	if err != nil {
 		return Time[T]{}, err
 	}
-	return Time[T]{tm: tm}, nil
+	return In[T](tm), nil
```

Comment:
I made adjustments due to the issue of varying time zones depending on the parsing content.